src/Discord.Net.Rest/Entities/Guilds/RestGuild.cs

```diff
@@ -1071,7 +1071,7 @@ public async Task<CustomSticker> CreateStickerAsync(string name, Image image, IE
         public Task<CustomSticker> CreateStickerAsync(string name,  string path, IEnumerable<string> tags, string description = null,
             RequestOptions options = null)
         {
-            var fs = File.OpenRead(path);
+            using var fs = File.OpenRead(path);
             return CreateStickerAsync(name, fs, Path.GetFileName(fs.Name), tags,  description,options);
```

Comment:
```suggestion
            return await CreateStickerAsync(name, fs, Path.GetFileName(fs.Name), tags,  description,options);
```
For correct dispose of file object from previous line.

Reply:
Thank you, and good catch. The other file needed it too.